test/Microsoft.AspNet.Mvc.Razor.Host.Test/MvcRazorHostTest.cs

```diff
@@ -254,6 +286,23 @@ private static Stream GetResourceStream(string resourceName)
                 generatedLocation: new MappingLocation(generatedLocation, contentLength));
         }
 
+        private class PathValidatingChunkInheritanceUtility : ChunkInheritanceUtility
```

Comment:
K, no big concenrs about the testability changes, just wanted to suggest an idea.